designer/server/src/main/scala/pl/touk/nussknacker/ui/process/ScenarioWithDetailsConversions.scala

```diff
@@ -55,6 +57,25 @@ object ScenarioWithDetailsConversions {
     )
   }
 
+  def skipAdditionalFields(details: ScenarioWithDetails, skipOptions: SkipAdditionalFields): ScenarioWithDetails = {
```

Comment:
I would rather prefer do not fetch this data from the DB at all. On the other hand I'm not sure is it now possible?